services/fleet.js

```diff
@@ -20,6 +20,30 @@ async function fleetApiGetRequest(uri) {
     return response
 };
 
+async function buildDashboard(){
+    // get all endpoints in order to iterate their ids in getScriptByEndpoint
+    endpoints = await listEndpoints();
+    // sending the endpoints to the 
+    data = getScriptByEndpoint(endpoints)
+    return data;
+};
+
+async function getScriptByEndpoint(endpointList){
+    // put in loop over endpointList
+    // change id to endpoint id
+    let scriptUri = 'GET /api/v1/fleet/hosts/:id/scripts'
+    // merge all api answers
+    // return all the data
+};
```

Comment:
This is not finished yet right?

Reply:
Yes, template before adding the actual code